definitions/npm/opentok_v2.8.x/flow_v0.61.x-/opentok_v2.8.x.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+declare module 'opentok' {
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#createSession
+  declare export type SessionOptions = {|
+    location?: string,
+    mediaMode?: 'routed' | 'relayed',
+    archiveMode?: 'always' | 'manual'
+  |};
+
+  // https://tokbox.com/developer/sdks/node/reference/Session.html
+  declare export type Session = {
+    sessionId: string
+  };
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#generateToken
+  declare export type GenerateTokenOptions = {|
+    role?: 'subscriber' | 'publisher' | 'moderator',
+    expireTime?: number,
+    data?: string,
+    initialLayoutClassList?: Array<string>
+  |};
+
+  declare export type Layout = {|
+    type: string,
+    stylesheet?: string
+  |};
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#setStreamClassLists
+  declare export type StreamLayoutClassItem = {|
+    id: string,
+    layoutClassList?: Array<string>
+  |};
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#getArchive
+  declare export type ListArchivesOptions = {|
+    count?: number,
+    offset?: number,
+    sessionId?: string
+  |};
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#startArchive
+  declare export type ArchiveOutputMode = 'composed' | 'individual';
+  declare export type StartArchiveOptions = {|
+    name?: string,
+    hasAudio?: boolean,
+    hasVideo?: boolean,
+    outputMode?: ArchiveOutputMode,
+    layout?: Layout
+  |};
+
+  // https://tokbox.com/developer/sdks/node/reference/OpenTok.html#dial
+  declare export type DialOptions = {|
+    headers?: {[string]: any},
+    auth?: {
+      username: string,
+      password: string
+    },
+    secure?: boolean
+  |};
+  declare export type SipInterconnect = {
+    id: string,
+    connectionId: string,
+    streamId: string
+  };
+
+  // https://tokbox.com/developer/sdks/node/reference/Archive.html
+  declare export class Archive {
+    createdAt: number;
+    duration: string;
+    id: string;
+    name: string;
+    partnerId: string;
+    reason: string;
+    sessionId: string;
+    size: number;
+    status: 'available' | 'expired' | 'failed' | 'paused' | 'started' | 'stopped' | 'uploaded';
+    hasAudio: boolean;
+    hasVideo: boolean;
+    outputMode: ArchiveOutputMode;
+    url: ?string;
+    delete(callback: (?Error) => any): void;
+    stop(callback: (?Error, ?Archive) => any): void;
+  }
+
+  // https://tokbox.com/developer/sdks/node/reference/Broadcast.html
+  declare export class Broadcast {
+    createdAt: number;
+    stop(callback: (?Error, ?Broadcast) => any): void;
+  }
+  declare export type BroadcastRtmpOutput = {|
+    id?: string,
+    serverUrl: string,
+    streamName: string,
+  |};
+  declare export type StartBroadcastOptions = {|
+    outputs: {
+      hls?: Object,
```

Comment:
`Object` is `any` from Flow 0.75 on. `{}` or `{ +[string]: mixed }` are alternatives.

Reply:
Fixed this with `{}`.